packages/main/src/request/index.ts

```diff
@@ -72,6 +72,20 @@ export const { actions, perform } = combineHandlers({
     })
   },
 
+  append(
+    connection: Connection,
+    flags: string[],
+    msgData: Buffer,
+    box: BoxSpecifier
```

Comment:
For consistency with other handlers please put the `box` argument order second in the list, after `connection`.